.github/workflows/units-tests-reusable.yml

```diff
@@ -69,4 +69,4 @@ jobs:
         if: ${{ inputs.collectCoverage == 'true' }}
         uses: codecov/codecov-action@v3
         with:
-          files: packages/@n8n/chat/coverage/cobertura-coverage.xml,packages/@n8n/nodes-langchain/coverage/cobertura-coverage.xml,packages/@n8n/permissions/coverage/cobertura-coverage.xml,packages/@n8n/client-oauth2/coverage/cobertura-coverage.xml,packages/cli/coverage/cobertura-coverage.xml,packages/core/coverage/cobertura-coverage.xml,packages/design-system/coverage/cobertura-coverage.xml,packages/editor-ui/coverage/cobertura-coverage.xml,packages/nodes-base/coverage/cobertura-coverage.xml,packages/workflow/coverage/cobertura-coverage.xml
+          files: packages/@n8n/chat/coverage/cobertura-coverage.xml,packages/@n8n/nodes-langchain/coverage/cobertura-coverage.xml,packages/@n8n/permissions/coverage/cobertura-coverage.xml,packages/@n8n/client-oauth2/coverage/cobertura-coverage.xml,packages/cli/coverage/cobertura-coverage.xml,packages/core/coverage/cobertura-coverage.xml,packages/design-system/coverage/cobertura-coverage.xml,packages/@n8n/codemirror-lang/coverage/cobertura-coverage.xml,packages/editor-ui/coverage/cobertura-coverage.xml,packages/nodes-base/coverage/cobertura-coverage.xml,packages/workflow/coverage/cobertura-coverage.xml
```

Comment:
Reminder for me to double check tomorrow if there is a way to avoid so much duplication.

Reply:
`codecov/codecov-action@v3` [does not support globs](https://github.com/codecov/codecov-action/issues/770), so we'd have to fork it. We could also dynamically construct the string and pass it via `$GITHUB_ENV`.

Either way, out of scope.

Reply:
Yeah, this is something we have discussed [in the past](https://github.com/n8n-io/n8n/pull/5679#discussion_r1135260289). Glob support will definitely be nice as we add more packages.